src/UXClient/Components/DateTimeButtonSingle/DateTimeButtonSingle.ts

```diff
@@ -37,17 +36,6 @@ class DateTimeButtonSingle extends DateTimeButton {
 
         let targetElement = <any>d3.select(this.renderTarget);
         var dateTimeTextChildren = (targetElement.select(".tsi-dateTimePickerContainer")).selectAll("*");
-        d3.select("html").on("click." + Utils.guid(), () => {
```

Comment:
why is this no longer needed?

Reply:
Same reason as the DateTimeButtonRange outer click functionality.